src/lamp_py/tableau/jobs/glides.py

```diff
@@ -190,15 +190,20 @@ def create_parquet(self, _: None) -> None:
         self.update_parquet(None)
 
     def update_parquet(self, _: None) -> bool:
-        # only run once per day after 11AM UTC - 6 or 7 AM EST
+        if self.first_run:  # type: ignore
```

Comment:
This Class is newly instantiated on every call to `start_parquet_updates` so this will always be True??